kumascript/macros/APIRef.ejs

```diff
@@ -159,37 +155,45 @@ var badges = {
   ObsoleteBadge : await template("ObsoleteBadge"),
 }
 
+function getPageTitle(page) {
+  if (page.shortTitle) {
+    return page.shortTitle;
+  }
+  else {
```

Comment:
No need to make this `else` explicit, since the `if` block returns. This saves us one indentation on the following lines.